Author: Simon Frankenberger <simon-alpine@fraho.eu>
Subject: Fix java version check

The java version check only detects java 17, 18, 19
and does not work with java 20+. This patch fixes the regex.

--- old/build/build.xml
+++ new/build/build.xml
@@ -174,7 +174,7 @@
     <target name="-check-java-version?">
         <echo></echo>
         <condition property="check-java-version.passed">
-            <matches pattern="1[789]+.*" string="${ant.java.version}"/>
+            <matches pattern="(1[789]+|2[0-9]+).*" string="${ant.java.version}"/>
         </condition>
         <fail unless="check-java-version.passed">
 .${line.separator}${line.separator}
